src/ShapeCrawler/ShapeCollection/SlideShapes.cs

```diff
@@ -582,8 +617,10 @@ private void AddPictureSvg(SvgDocument image)
         // Fix up the sizes
         var xEmu = UnitConverter.HorizontalPixelToEmu(100);
         var yEmu = UnitConverter.VerticalPixelToEmu(100);
-        var cxEmu = UnitConverter.HorizontalPixelToEmu(width);
-        var cyEmu = UnitConverter.VerticalPixelToEmu(height);
+
+        // TODO: Refactor this all to decimal when that PR lands
```

Comment:
If I'm not mistaken, this is about “X” PR https://github.com/ShapeCrawler/ShapeCrawler/pull/688 ?

Reply:
Right. When `HorizontalPixelToEmu()` changes over to decimal, this will need a little cleanup of casts here.

Reply:
This is all updated now